src/mame/video/hng64_3d.hxx

```diff
@@ -612,6 +618,9 @@ void hng64_state::recoverPolygonBlock(const uint16_t* packet, int& numPolys)
 					currentPoly.vert[m].normal[1] = uToF(chunkOffset[10 + (9*m)]);
 					currentPoly.vert[m].normal[2] = uToF(chunkOffset[11 + (9*m)]);
 					currentPoly.vert[m].normal[3] = 0.0f;
+
+					if ( currentPoly.flatShade )
```

Comment:
Please make the code consistent – don’t put extra spaces inside the parentheses like this.